lib/seed/Seeder.js

```diff
@@ -87,7 +96,11 @@ class Seeder {
   }
 
   _getNewStubFilePath(name) {
-    return path.join(this._absoluteConfigDir(), this._getNewStubFileName(name));
+    const fileName = this._getNewStubFileName(name);
+    const dirs = this._absoluteConfigDirs();
+    const dir = dirs.slice(-1)[0]; // Get last specified directory
+    return path.join(dir, fileName);
+    // return path.join(this._absoluteConfigDir(), this._getNewStubFileName(name));
```

Comment:
should this be commented out?

Reply:
This is fixed in the latest commit. Do I let you resolve the conversation ?